compiler-core/src/build.rs

```diff
@@ -98,9 +98,14 @@ pub struct Package {
 }
 
 impl Package {
-    pub fn attach_doc_and_module_comments(&mut self) {
+    pub fn attach_doc_and_module_comments(&mut self, hidden_modules: &Vec<String>) {
         for mut module in &mut self.modules {
-            module.attach_doc_and_module_comments();
+            if !hidden_modules
+                .iter()
+                .any(|pattern| module.name.starts_with(pattern))
```

Comment:
Does this work on Windows?

Would `one_two` get ignored by a pattern of `one`? I think we likely want to make the globbing explicit using git-like `one/*` syntax.